srcpkgs/mpv/template

```diff
@@ -24,12 +25,19 @@ hostmakedepends="pkg-config python3-docutils $(vopt_if wayland wayland-devel)"
 makedepends="MesaLib-devel ffmpeg6-devel harfbuzz-devel libXv-devel libass-devel
  libbluray-devel libcdio-paranoia-devel libdvdnav-devel libuuid-devel
  libva-devel rubberband-devel uchardet-devel libplacebo-devel libarchive-devel
- $(vopt_if alsa alsa-lib-devel) $(vopt_if caca libcaca-devel)
- $(vopt_if jack jack-devel) $(vopt_if lcms lcms2-devel)
- $(vopt_if lua lua52-devel) $(vopt_if openal libopenal-devel)
- $(vopt_if pulseaudio pulseaudio-devel) $(vopt_if pipewire pipewire-devel)
- $(vopt_if sdl2 SDL2-devel) $(vopt_if vapoursynth vapoursynth-devel)
- $(vopt_if vdpau libvdpau-devel) $(vopt_if sndio sndio-devel)
+ $(vopt_if alsa alsa-lib-devel)
+ $(vopt_if caca libcaca-devel)
+ $(vopt_if jack jack-devel)
+ $(vopt_if javascript mujs-devel)
+ $(vopt_if lcms lcms2-devel)
+ $(vopt_if lua lua52-devel)
+ $(vopt_if openal libopenal-devel)
+ $(vopt_if pulseaudio pulseaudio-devel)
+ $(vopt_if pipewire pipewire-devel)
+ $(vopt_if sdl2 SDL2-devel)
+ $(vopt_if vapoursynth vapoursynth-devel)
+ $(vopt_if vdpau libvdpau-devel)
+ $(vopt_if sndio sndio-devel)
```

Comment:
please don't separate these out to one per line, it just obfuscates history/blame